nodeadm/internal/api/featuregates.go

```diff
@@ -0,0 +1,32 @@
+//go:build !ignore_autogenerated
```

Comment:
Got this error form build after moving the feature gate to internal/api package.
```
/Volumes/workplace/githubForks/amazon-eks-ami/nodeadm/internal/api/featuregates.go:8:2: invalid field type: func(github.com/awslabs/amazon-eks-ami/nodeadm/internal/api.Feature, map[github.com/awslabs/amazon-eks-ami/nodeadm/internal/api.Feature]bool) bool
```
Thought this can be workaround by ignoring the featuregate file, by adding , but not sure if this is the correct way